modules/using-cluster-compare-telco-ref.adoc

```diff
@@ -0,0 +1,152 @@
+:_mod-docs-content-type: PROCEDURE
+[id="using-cluster-compare-telco_ref_{context}"]
+= Example: Comparing a cluster with the telco core reference configuration
+
+You can use the `cluster-compare` plugin to compare a reference configuration with a configuration from a live cluster or `must-gather` data. 
+
+This example compares a configuration from a live cluster with the telco core reference configuration. The telco core reference configuration is derived from the telco core reference design specifications (RDS). The telco core RDS is designed for clusters to support large scale telco applications including control plane and some centralized data plane functions.
+
+The reference configuration is packaged in a container image with the telco core RDS. 
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You have credentials to access the `registry.redhat.io` container image registry.
+
+.Procedure
+
+. Login to the container image registry with your credentials by running the following command:
++
+[source,terminal]
+----
+$ podman login registry.redhat.io
+----
+
+. Extract the content from the `telco-core-rds-rhel9` container image by running the following commands:
++
+[source,terminal]
+----
+$ mkdir -p ./out
+----
++
+[source,terminal]
+----
+$ podman run -it registry.redhat.io/openshift4/openshift-telco-core-rds-rhel9:v4.17 | base64 -d | tar xv -C out
+----
++
+You can view the reference configuration in the `reference-crs-kube-compare/` directory.
++
+[source,text]
+----
+out/telco-core-rds/configuration/reference-crs-kube-compare/
+├── metadata.yaml <1>
+├── optional <2>
+│   ├── logging
+│   ├── networking
+│   ├── other
+│   └── tuning
+└── required <3>
+    ├── networking
+    ├── other
+    ├── performance
+    ├── scheduling
+    └── storage
+----
+<1> Configuration file for the reference configuration.
+<2> Directory for optional templates.
+<3> Directory for required templates.
+
+. Compare the configuration for your cluster to the telco core reference configuration by running the following command:
++
+[source,terminal]
+----
+$ oc cluster-compare -r out/telco-core-rds/configuration/reference-crs-kube-compare/metadata.yaml
+----
++
+.Example output
+[source,terminal]
+----
```

Comment:
@lack - I've added a snippet of example output from a clusterbot. Would it be better to show more typical output from a cluster with a telco core profile applied? I don't have access to such a cluster...

Reply:
I think that's fine unless we can come up with something better.  I'll see if I can find a real cluster with the core config.